### PR TITLE
docs: update taosBenchmark doc

### PR DESCRIPTION
**Pull request by zhaoyanggh**

### CODE REVIEW

**Review by wu-champion — APPROVED** [review]
LGTM

File: docs/zh/14-reference/05-taosbenchmark.md [hunk 1]
```diff
@@ -227,45 +227,34 @@ taosBenchmark -A INT,DOUBLE,NCHAR,BINARY\(16\)
...
+- **stream_name** : 流式计算的名称，必填项。
 
-- **blocks** : 每个 vnode 中缓存块的数量，默认为 6。
+- **stream_stb** : 流式计算对应的超级表名称，必填项。
 
-- **precision** : 数据库时间精度，默认值为 "ms"。
+- **stream_sql** : 流式计算的sql语句，必填项。
 
-- **keep** : 保留数据的天数，默认值为 3650。
+- **trigger_mode** : 流式计算的触发模式，可选项。
```
Comment (jiajingbin) [inline docs/zh/14-reference/05-taosbenchmark.md:246]: trigger_mode must be "at_once" "window_close" "max_delay **s". max_delay unit(**s) could be d/m or others supported by TDengine

File: docs/zh/14-reference/05-taosbenchmark.md [hunk 2]
```diff
@@ -227,45 +227,34 @@ taosBenchmark -A INT,DOUBLE,NCHAR,BINARY\(16\)
...
 
-- **minRows** : 文件块中的最小记录数，默认值为 100。
+- **watermark** : 流式计算的水印，可选项。
```
Comment (jiajingbin) [inline docs/zh/14-reference/05-taosbenchmark.md:248]: The watermark field is a string, which must be added with a time unit

File: docs/zh/14-reference/05-taosbenchmark.md (hunk 1 above)
Comment (zhaoyanggh) [reply docs/zh/14-reference/05-taosbenchmark.md:246]: I will make a link to the stream official page

**Review by sangshuduo — APPROVED** [review]
talked to yang, he will update missed part later. merge this PR now.

**Pull request merged**